java/src/org/openqa/selenium/firefox/FirefoxDriver.java

```diff
@@ -283,39 +285,27 @@ public DevTools getDevTools() {
       .orElseThrow(() -> new DevToolsException("Unable to initialize CDP connection"));
   }
 
-  @Override
-  public Optional<BiDi> maybeGetBiDi() {
-    if (biDi != null) {
-      return Optional.of(biDi);
-    }
-
+  private Optional<BiDi> createBiDi(Optional<URI> biDiUri) {
     if (!biDiUri.isPresent()) {
       return Optional.empty();
     }
 
     URI wsUri = biDiUri.orElseThrow(
-      () -> new BiDiException("This version of Firefox or geckodriver does not support BiDi"));
+      () -> new BiDiException("This version of Chromium driver does not support BiDi"));
```

Comment:
Shouldn't the message be for firefox driver?

Reply:
Yes! Thank you.